gslib/utils/copy_helper.py

```diff
@@ -1612,7 +1617,11 @@ def _SetContentTypeFromFile(src_url, dst_obj_metadata):
               'Encountered OSError running "file -b --mime %s"\n%s' %
               (real_file_path, e))
       else:
-        content_type = mimetypes.guess_type(real_file_path)[0]
+        _, _, extension = real_file_path.rpartition('.')
```

Comment:
Nice use of rpartition!

Reply:
BTW, https://docs.python.org/3/library/os.path.html#os.path.splitext is commonly used to get the extension.